examples/box_game/box_game.rs

```diff
@@ -161,31 +161,34 @@ pub fn increase_frame_system(mut frame_count: ResMut<FrameCount>) {
 pub fn move_cube_system(
     mut query: Query<(&mut Transform, &mut Velocity, &Player), With<Rollback>>,
     inputs: Res<PlayerInputs<BoxConfig>>,
+    time: Res<Time>,
```

Comment:
I think we should either use `RollbackTime` here, or add a comment explaining that `Time` is safe to use.

Reply:
I've also added some documentation to `GgrsPlugin` explaining what parts of the ECS are being rollback managed by default.